src/extractors/custom/bookwalker.jp/index.js

```diff
@@ -0,0 +1,32 @@
+export const BookwalkerJpExtractor = {
+  domain: 'bookwalker.jp',
+
+  title: {
+    selectors: ['h1.main-heading'],
+  },
+
+  author: null,
```

Comment:
Was there any reason why the `.authors` selector was not used here?

Reply:
Because I think the `.authors` selector should be set the person who wrote this article.
Although the author of the novel is listed in this article, the author of the article is not listed.
So I did not use '.authors' selector.
Is this way of thinking wrong?

Reply:
Let's proceed with this change, as discussed in https://github.com/postlight/mercury-parser/pull/374#issuecomment-482359833